src/Application/BootstrapBuilder.php

```diff
@@ -64,14 +64,18 @@ public function getBootstrap() {
      *
      */
     protected function getApplicationConfigurations() {
-        $configurations = [];
-        $uriParameters = explode('/', trim($_SERVER['REQUEST_URI'], '/') . '/');
-        $appUriRequest = strpos($uriParameters[0], '.') === false ? $uriParameters[0] : $uriParameters[1];
+      $configurations = [];
+        $uriParameters = explode('?', trim($_SERVER['REQUEST_URI'], '/') );
 
+//      $appUriRequest = strpos($uriParameters[0], '.') === false ? $uriParameters[0] : $uriParameters[1];
         // @todo remove framework.php
+      // todo 强行修改
+      $appUriRequest = $uriParameters[0];
+
```

Comment:
Hi @ifandpromiss 

firstly, please use English for the comments in the projects, that are holed on Github. Secondly, you will never get the right URI by simply taking the first element in the array, if the URI contains index.php or index.html. For example http://www.abc.com/index.php/myuri. The best way to get the uri is try the code with regular expression